index.bs

```diff
@@ -4071,13 +4070,13 @@ even if the SafetyNet API is also present.
     as follows:
     - Verify that |attStmt| is valid CBOR conforming to the syntax defined above and perform CBOR decoding on it to extract the
         contained fields.
-    - Verify that |response| is a valid SafetyNet response of version |ver|.
+    - Verify that |response| is a valid SafetyNet response of version |ver| 
+        by following the steps indicated by the [SafetyNet online documentation](https://developer.android.com/training/safetynet/attestation.html#compat-check-response).
+        As of this writing, there is only one format of the SafetyNet response.  This step is only necessary if the SafetyNet
+        response becomes different for different Google Play Services versions.
     - Verify that the `nonce` attribute in the payload of |response| is identical to the Base64 encoding of the SHA-256 hash of the concatenation of |authenticatorData| and |clientDataHash|.
-    - Let |x5c| be the value of the `x5c` attribute in the header of |response|, parsed as a sequence of X.509 certificates.
-    - Let |attestationCert| be the first element of |x5c|.
-    - Verify that |attestationCert| is issued to the hostname "attest.android.com" (see
-        [SafetyNet online documentation](https://developer.android.com/training/safetynet/index.html#compat-check-response)).
-    - Verify that the `ctsProfileMatch` attribute in the payload of |response| is [TRUE].
+    - Verify that the SafetyNet response actually came from the SafetyNet service by following steps in the 
```

Comment:
I still think there's a "the" missing in "following steps" here, but I guess it's also fine enough to leave as is.